bundler/lib/bundler/dsl.rb

```diff
@@ -488,6 +489,16 @@ def normalize_source(source)
       end
     end
 
+    def deprecate_legacy_windows_platforms(platforms)
+      dep_platforms = Array(platforms).select {|pl| pl.to_s.match?(/mingw|mswin/) }
+      return if dep_platforms.empty?
+
+      dep_platforms = dep_platforms.map! {|pl| ":#{pl}" }.join(", ")
+      message = "Platform #{dep_platforms} is deprecated. Please use platform :windows instead."
+      removed_message = "Platform #{dep_platforms} has been removed. Please use platform :windows instead."
+      Bundler::SharedHelpers.major_deprecation 3, message, removed_message: removed_message
```

Comment:
This should be 2, rather than 3, to make the deprecation effective immediately.

Reply:
Done.